src/main/java/com/michalszekalski/bootcamp_zad26/Bet.java

```diff
@@ -0,0 +1,75 @@
+package com.michalszekalski.bootcamp_zad26;
+
+import jakarta.persistence.*;
+
+@Entity
+public class Bet {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private long betValue;
+
+    private String typerName;
+
+    @Enumerated(EnumType.STRING)
+    private Winner winner;
+
+    @ManyToOne
+    private Match match;
+
+    private String idString;
```

Comment:
Albo bym tutaj pomyślał nad inną nazwą, albo skoro to pole też musi być unikalne, to zrobiłbym z niego też id na poziomie bazy danych